MANIFEST.in

```diff
@@ -14,8 +14,6 @@ graft requirements
 graft tests
 global-exclude *.pyc
 global-exclude *.cache
-exclude propcache/*.c
-exclude propcache/*.html
-exclude propcache/*.so
-exclude propcache/*.pyd
```

Comment:
To summarize https://github.com/aio-libs/propcache/pull/29/files#r1789278897, keep excluding these, just adjust the paths.

Reply:
@bdraco I meant that the so/pyd lines should remain in the manifest file